img/filter.svg

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
We should minify this one with https://github.com/nextcloud/server/blob/master/build/image-optimization.sh#L10

Reply:
ok, what is exactly to do?